src/Microsoft.DotNet.Interactive.Tests/Utility/DirectoryUtility.cs

```diff
@@ -58,5 +58,12 @@ public static class DirectoryUtility
                     t.content);
             }
         }
+
+        public static string GetPathToSrcDirectory()
+        {
+            string pathAssembly = System.Reflection.Assembly.GetExecutingAssembly().Location;
+            string folderAssembly = Path.GetDirectoryName(pathAssembly);            
+            return Path.Combine(folderAssembly, "..", "..", "..", "..", "..", "src");
```

Comment:
I'd feel more comfortable if the .csx/.fsx files were added as `<Content>` in the .csproj and copied to the output directory, instead.  With this layout anybody that builds and tests with the env var `%DISABLEARCADE%=1` will be broken, aka, @colombod and @jonsequitur.

Reply:
Or better, just emit the file during the test run, so the total code for the test case is more cohesive.

Reply:
please have a look at this: https://github.com/dotnet/interactive/pull/1176/commits/433e8f6991745e7a7ffe7ba4d388de27f4fc506c